iwyu_output.cc

```diff
@@ -275,6 +275,22 @@ OneUse::OneUse(const string& symbol_name, const FileEntry* dfn_file,
     suggested_header_ = decl_filepath_;
 }
 
+OneUse::OneUse(const FileEntry* dfn_file, const string& dfn_filepath)
```

Comment:
I'm a little confused why this new overload is necessary. I played around with it on my local machine, but I didn't get my head around it.

Are there two changes in this commit -- one to use `FileEntry` and one to simplify construction at https://github.com/include-what-you-use/include-what-you-use/pull/911/files#diff-96c9906a158b909fda0f001cb5bd04615d46704df24173306894f3d228abbed1R669?

This one looks kind-of redundant, but I'm not entirely sure what's going on. One thing at a time is my motto :-)

Reply:
Thanks for pointing that out, I actually had mixed feelings about adding another overload with less arguments. Reverted the change and reused the existing ctor.